waku/node/v2/wakunode2.nim

```diff
@@ -129,13 +119,7 @@ proc start*(node: WakuNode) {.async.} =
   node.switch.mount(node.wakuFilter)
   node.subscriptions.subscribe(WakuFilterCodec, node.wakuFilter.subscription())
 
-  proc relayHandler(topic: string, data: seq[byte]) {.async, gcsafe.} =
-    let msg = WakuMessage.init(data)
-    if msg.isOk():
-      node.filters.notify(msg.value(), "")
-      await node.subscriptions.notify(topic, msg.value())
-
-  await node.wakuRelay.subscribe("waku", relayHandler)
+  await sleepAsync(5.seconds)
```

Comment:
Comment on why we are sleeping and under what conditions it can be removed. No one knows what this is for otherwise.